File: lab2_fabrica/fabrica/Mask.cs
using System;
using System.Drawing;
using System.Runtime.Serialization;
using System.Text;

namespace fabrica
{
    [DataContract]
    public class Mask: SkinCare
    {
        [DataMember]
        private int time;
        [DataMember]
        private String maskType;
        public int Time
        {
            get { return time; }
            set {time = value; }
        }
        public String MaskType
        {
            get { return maskType; }
            set { maskType = value; }
        }

        public Mask(String name, String brand, double price, String activeComponent, int time, String maskType) : base(name, brand, price, activeComponent)
        {
            this.maskType = maskType;
            this.time = time;
        }

        public override string getinfo()
        {
            String info =
            "Name: " + name + "\r\n" +
            "Brand: " + brand + "\r\n" +
            "Price: " + price + "$" + "\r\n" +
            "Type: " + maskType + "\r\n" +
            "Active Component: " + activeComponent + "\r\n" +
            "Time to apply: " + time + " мин \r\n";
            return info;
        }

        public override Bitmap Img()
        {
            return (Bitmap)Bitmap.FromFile(".//img//mask.png");
        }
    }
}

File: lab1 class/untitled/src/Mask.java
public class Mask extends SkinCare{
    private int time;
    private String maskType;
    private boolean removable ;
    protected Mask( String name, String brand, double price, String activeComponent, String scent, int time, String maskType, boolean removable) {
        super( name, brand, price, activeComponent, scent);
        this.maskType=maskType;
        this.time = time;
        this.removable = removable;
    }

    public void remove(){
        System.out.println("Removing " + name + " ...");
    };
    @Override
    public void printinfo() {
        System.out.println(name);
        System.out.println("Brand: "+ brand);
        System.out.println("Price: "+ price + "$ ");
        System.out.println("Type: " + maskType);
        System.out.println("Scent: " + scent);
        System.out.println("Apply for " + time + " minutes");
        if (removable ){
            System.out.println("Wash your face with worm water after \n");}
        else {
            System.out.println("");
        }

    }
}
